src/components/PageSectionHeader/PageSectionHeader.tsx

```diff
@@ -0,0 +1,23 @@
+import { Typography } from "@material-ui/core";
+import VerticalSpacer from "@saleor/apps/components/VerticalSpacer";
+import React from "react";
+
+interface PageSectionHeaderProps {
+  title?: React.ReactNode;
+  description?: React.ReactNode;
```

Comment:
Why are these typed ReactNode? Do we have any cases where titles or descriptions are not strings?

Reply:
Nope, changed 👍